lib/Frontend/Frontend.cpp

```diff
@@ -807,6 +807,20 @@ void CompilerInstance::parseAndCheckTypesUpTo(
                         options.WarnLongExpressionTypeChecking,
                         options.SolverExpressionTimeThreshold,
                         options.SwitchCheckingInvocationThreshold);
+
+    const std::string filename = llvm::sys::path::stem(SF.getFilename()).str();
```

Comment:
There's no real reason not leave this as a StringRef, instead of forcing the copy. But it looks like it's actually not used at all?

Reply:
That should be removed. That's a leftover line from a temporary workaround.